app/tests/conftest.py

```diff
@@ -5,6 +5,7 @@
 from app.services.auth import OperatorAuthHandler
 
 from . import client
+from .helpers import *  # noqa: F403, F401
```

Comment:
Don't use wildcard import.

Reply:
Let this one slide please senpai :v, We need all the fixtures to be imported and wildcard is fastest

Reply:
Then why don't you just put them directly inside `conftest.py`?

Reply:
I dont wanna bloat the whole file since these fixtures are more `service-specific`. Wanna keep it clean ya know :v